tests/v0_tests/test_model_cache_management.py

```diff
@@ -0,0 +1,53 @@
+import copy
+import math
+import pprint
+import random
+import requests
+from marqo.client import Client
+from marqo.errors import MarqoApiError, MarqoError, MarqoWebError
+from tests.marqo_test import MarqoTestCase
+from marqo import enums
+from unittest import mock
+
+
+class TestModelCacheManagement(MarqoTestCase):
+
+    # NOTE: The cuda should already have model loaded in the startup
+    def setUp(self) -> None:
+        self.client = Client(**self.client_settings)
+        self.index_name = "test_index"
+        self.MODEL = "ViT-L/14"
```

Comment:
Can we set this to the smaller CLIP model? Sometimes test instances can't handle the memory footprint of this large model

Reply:
we change to "ViT-B/32" .